.github/workflows/sync-branch.yml

```diff
@@ -35,4 +35,4 @@ jobs:
           echo "$(pwd)" >> $GITHUB_PATH
       
       - name: Run merge command
-        run: flub merge branches -a ${{ secrets.BOT_MAIN_NEXT_WORKFLOW_PAT }} -s $SOURCE_BRANCH -t $TARGET_BRANCH -b 10
+        run: flub merge branches -a ${{ secrets.BOT_MAIN_NEXT_WORKFLOW_PAT }} -s $SOURCE_BRANCH -t $TARGET_BRANCH -b 10 --reviewers sonalideshpandemsft --reviewers tylerbutler --reviewers scottn12
```

Comment:
Does this need to be hardcoded? Can it be a group instead, that we can assign members to?